src/Shared/FrameworkLocationHelper.cs

```diff
@@ -373,6 +378,9 @@ internal static class FrameworkLocationHelper
             { (dotNetFrameworkVersion471, visualStudioVersion160), (dotNetFrameworkVersion47, visualStudioVersion160) },
             { (dotNetFrameworkVersion472, visualStudioVersion160), (dotNetFrameworkVersion471, visualStudioVersion160) },
             { (dotNetFrameworkVersion48, visualStudioVersion160), (dotNetFrameworkVersion472, visualStudioVersion160) },
+
+            // VS 17
+            { (dotNetFrameworkVersion481, visualStudioVersion170), (dotNetFrameworkVersion48, visualStudioVersion170) },
```

Comment:
Should this be a full list of all the supported targets for VS 17? That's my naive interpretaion of the VS16 section above.

Reply:
I was focusing a bit too much on net481 specifically here. While we're here we should include all versions compatible with vs 17